### ISSUE TITLE
Unable to setup Payment system from new install

### ISSUE THREAD

**Issue by hermbo** [sharetribe/sharetribe#3969]
Just fetched a new copy and loaded it with a new database only to see the Payments section missing. Anyone else having this issue?

Commit: 4818252
Environment: development
node version: v10.15.3
npm version: 6.4.1

### No payments system in the admin
![Screen Shot 2020-01-17 at 2 57 55 PM](https://user-images.githubusercontent.com/38641518/72651930-e9674600-3939-11ea-8981-b1af6c59471c.png)

### When going to it manually getting this error
`/en/admin/payment_preferences`
![Screen Shot 2020-01-17 at 2 58 12 PM](https://user-images.githubusercontent.com/38641518/72651931-e9ffdc80-3939-11ea-9610-86812e9e2e0e.png)

### When trying to edit a listing that has payment
![Screen Shot 2020-01-17 at 3 08 53 PM](https://user-images.githubusercontent.com/38641518/72652358-516a5c00-393b-11ea-98df-14238166b294.png)

**Comment by thomasmalbaux** [comment]
Hi there!

Unfortunately I'm not able to reproduce that issue. Did you have a chance to follow the instructions at https://github.com/sharetribe/sharetribe#payments?

Don't hesitate to share your specific situation with other users of the Community Edition at https://www.sharetribe.com/community/, there might be already a thread about a similar situation!

**Comment by jeremyevans6** [comment]
I also get this issue with a fresh install in the current branch. I've also solved it by using the posted instructions for Stripe. I'd like to share my experience here since I'm trawling these issues myself and would like to provide some more details for fools such as myself.

/en/admin/payment_preferences redirects to en/admin/details/edit with the payments_not_enabled error.

This error is also complaining because of a lack of English translation in en.admin.communities.settings, so I added a field. This corrected the error, which now displays my chosen text, but this did nothing to remedy the inability to access the payments' settings. I won't go into more details here since this doesn't seem to contribute to the resolution of the issue, though it probably should be patched in the next build.

**The fix**: 
> `If Stripe isn't automatically enabled in the admin panel after upgrading to 7.2.0, you should run the following commands in your Rails console, where <ID> is your marketplace ID (probably 1): TransactionService::API::Api.processes.create(community_id: <ID>, process: :preauthorize, author_is_seller: true) and TransactionService::API::Api.settings.provision(community_id: <ID>, payment_gateway: :stripe, payment_process: :preauthorize, active: true).`

In other words, I opened a rails console in a new terminal while running my server and delayed jobs worker. In the Sharetribe directory, I ran this:
$ `rails console`
$ `TransactionService::API::Api.processes.create(community_id: 1, process: :preauthorize, author_is_seller: true)`
$ `TransactionService::API::Api.settings.provision(community_id: 1, payment_gateway: :stripe, payment_process: :preauthorize, active: true)`

After this, the error was removed and I could access the payment settings normally.

Sharetribe Version:
53703a2
Environment:
Ubuntu 20.04
Node: 10.15.3
NPM: 6.4.1
Server mode: Development
Fresh install, following posted instructions
